osgp/platform/osgp-secret-management/src/main/java/org/opensmartgridplatform/secretmanagement/application/config/SecretManagementInitializer.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2020 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.secretmanagement.application.config;
+
+import ch.qos.logback.core.joran.spi.JoranException;
+import ch.qos.logback.ext.spring.LogbackConfigurer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.WebApplicationInitializer;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+
+public class SecretManagementInitializer implements WebApplicationInitializer {
```

Comment:
can you explain why is this class needed? either the logback.xml from the classpath is loaded or one is provided via the context.xml.

Is there a reason we use JDNI here? is the log file neede by JMX ?